upb/json/parser.rl

```diff
@@ -1036,11 +1039,13 @@ static bool end_membername(upb_json_parser *p) {
 
       return true;
     } else {
-      /* TODO(haberman): Ignore unknown fields if requested/configured to do
-       * so. */
-      upb_status_seterrf(&p->status, "No such field: %.*s\n", (int)len, buf);
-      upb_env_reporterror(p->env, &p->status);
-      return false;
+      if (p->method->ignore_unknown_fields) {
+        return false;
```

Comment:
I think you forget to call ```multipart_end```

Reply:
Have you tested this can ignore unknown?

Reply:
This was tested as part of the original PR via the ruby interface. Is there a test suite here I can leverage?

https://github.com/google/protobuf/pull/4304/files

What does `multipart_end` do and why do I need to call it here?